maven-surefire-common/src/main/java/org/apache/maven/plugin/surefire/AbstractSurefireMojo.java

```diff
@@ -281,6 +285,21 @@ public abstract class AbstractSurefireMojo extends AbstractMojo implements Suref
     @Parameter(property = "maven.test.additionalClasspath")
     private String[] additionalClasspathElements;
 
+    /**
+     * Additional Maven dependencies to be used in the test execution classpath.
+     * Each element supports the parametrization like documented in <a href="https://maven.apache.org/pom.html#dependencies">POM Reference: Dependencies</a>.
+     * <p>
+     * Those dependencies are automatically collected (i.e. have their full dependency tree calculated) and then all underlying artifacts are resolved from the repository (including their transitive dependencies).
+     * Afterwards the resolved artifacts are filtered to only contain {@code compile} and {@code runtime} scoped ones and appended to the test execution classpath
+     * (after the ones from {@link #additionalClasspathElements}).
+     * <p>
+     * The dependency management from the project is not taken into account.
+     *
+     * @since 3.2
+     */
+    @Parameter(property = "maven.test.additionalClasspathDependencies")
+    private Dependency[] additionalClasspathDependencies;
```

Comment:
`List<Dependency>`? 
with empty list as default - we will not need check for null value

Reply:
This is just to be more in line with existing parameters. But I can switch to `List`. The `null` check is not necessary for either array nor Collection, as plexus.inject will inject an empty array.